src/drivers/sdl/lv_sdl_window.c

```diff
@@ -196,12 +196,13 @@ static void flush_cb(lv_display_t * disp, const lv_area_t * area, uint8_t * px_m
         int32_t y;
         uint8_t * fb_tmp = dsc->fb_act;
         uint32_t px_size = lv_color_format_get_size(lv_display_get_color_format(disp));
-        uint32_t px_map_stride = lv_area_get_width(area) * px_size;
+        uint32_t px_map_stride = lv_draw_buf_width_to_stride(lv_area_get_width(area), lv_display_get_color_format(disp));
+        uint32_t data_size = lv_area_get_width(area) * px_size;
```

Comment:
`data_size` -> `area_stirde` ?

Reply:
But it's not a stride of any buffer. It's the valid data size contained in one line.

Reply:
OK